CodeStarterCamp_Week3/STEP2.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  STEP1.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by 한겨레 on 2022/07/08.
+//
+
+import Foundation
+
+class Person {
+    var name: String
+    var age: Int
+    var money: Int
+    
+    func order(coffee: Coffee, coffeeShop: CoffeeShop) {
+        if let coffeePrice = coffeeShop.menu[coffee] {
+            if coffeePrice > money {
+                let nonPrice = coffeePrice - money
+                print("잔액이 \(nonPrice)원만큼 부족합니다.")
+            }
+            else {
+                coffeeShop.make(coffee: coffee, from: self.name)
+                money -= coffeePrice
+            }
+        }
+    }
+    
+    init (name: String, age: Int, money: Int) {
+        self.name = name
+        self.age = age
+        self.money = money
+    }
```

Comment:
```suggestion
class Person {
    var name: String
    var age: Int
    var money: Int
        
    init (name: String, age: Int, money: Int) {
        self.name = name
        self.age = age
        self.money = money
    }
    
    func order(coffee: Coffee, coffeeShop: CoffeeShop) {
        if let coffeePrice = coffeeShop.menu[coffee] {
            if coffeePrice > money {
                let nonPrice = coffeePrice - money
                print("잔액이 \(nonPrice)원만큼 부족합니다.")
            }
            else {
                coffeeShop.make(coffee: coffee, from: self.name)
                money -= coffeePrice
            }
        }
    }
```

`init` 은 코드의 가독성을 위해 주로 프로퍼티 밑에 위치하는걸 선호합니다!